package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
This is the package lock file generated by npm under Node v16.

The CI job seems to have no issues with this, and it all seems to work correctly locally for me.